front/src/AuthenticationScreen/index.js

```diff
@@ -15,30 +15,38 @@ import ConnectionBar from "../BottomBar/ConnectionBar";
  *  - onSubmit: A function that will send the token to the SRRS
  *  - connectionStatus: The current connection status
  */
+
 class AuthenticationScreen extends Component {
   constructor(props) {
     super(props);
     this.state = {
       token: "",
-      showNotification: false
+      showNotification: false,
+      notification: "Incorrect token"
```

Comment:
why is this hardcoded? Shouldn't it be set a notification is created?

Reply:
we don't set the notification in the same place as where we receive the text. Therefore it has to be saved somewhere.

Reply:
You do set in `onIncorrectToken`, right?